br/br-snapshot-guide.md

```diff
@@ -0,0 +1,180 @@
+
+## Performance and impact
+
+### Performance and impact of snapshot backup
+
+The backup feature has some impact on cluster performance (transaction latency and QPS). However, you can mitigate the impact by adjusting the number of backup threads [`backup.num-threads`](/tikv-configuration-file.md#num-threads-1) or by adding more clusters.
+
+To illustrate the impact of backup, this document lists the test conclusions of several snapshot backup tests:
+
+- (5.3.0 and earlier) When the backup threads of `br` on a TiKV node take up 75% of the total CPU of the node, the QPS is reduced by 30% of the original QPS.
+- (5.4.0 and later) When there are no more than `8` threads of `br` on a TiKV node and the cluster's total CPU utilization does not exceed 80%, the impact of `br` tasks on the cluster (write and read) is 20% at most.
+- (5.4.0 and later) When there are no more than `8` threads of `br` on a TiKV node and the cluster's total CPU utilization does not exceed 75%, the impact of `br` tasks on the cluster (write and read) is 10% at most.
+- (5.4.0 and later) When there are no more than `8` threads of `br` on a TiKV node and the cluster's total CPU utilization does not exceed 60%, `br` tasks have little impact on the cluster (write and read).
+
+You can use the following methods to manually control the impact of backup tasks on cluster performance. However, these two methods also reduce the speed of backup tasks while reducing the impact of backup tasks on the cluster.
+
+- Use the `--ratelimit` parameter to limit the speed of backup tasks. Note that this parameter limits the speed of **saving backup files to external storage**. When calculating the total size of backup files, use the `backup data size(after compressed)` as a benchmark.
+- Adjust the TiKV configuration item [`backup.num-threads`](/tikv-configuration-file.md#num-threads-1) to limit the number of threads used by backup tasks. According to internal tests, when `br` uses no more than `8` threads for backup tasks, and the total CPU utilization of the cluster does not exceed 60%, the backup tasks have little impact on the cluster, regardless of the read and write workload.
+
+The impact of backup on cluster performance can be reduced by limiting the backup threads number, but this affects the backup performance. The preceding tests show that the backup speed is proportional to the number of backup threads. When the number of threads is small, the backup speed is about 20 MiB/thread. For example, 5 backup threads on a single TiKV node can reach a backup speed of 100 MiB/s.
+
+### Performance and impact of snapshot restore
+
+- During restoring data, TiDB tries to fully utilize the TiKV CPU, disk IO, and network bandwidth resources. Therefore, it is recommended to restore the backup data on an empty cluster to avoid affecting the running applications.
```

Comment:
```suggestion
- During data restore, TiDB tries to fully utilize the TiKV CPU, disk IO, and network bandwidth resources. Therefore, it is recommended to restore the backup data on an empty cluster to avoid affecting the running applications.
```